clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/data/task/jedis/JedisTaskRepository.groovy

```diff
@@ -124,6 +126,8 @@ class JedisTaskRepository implements TaskRepository {
       pipe.expire(resultId, TASK_TTL)
       pipe.sync()
     }
+
+    log.debug("addResultObjects (resultId: ${resultId}, values: ${values})")
```

Comment:
lose as in they are there then not there, or as in we see a complete task but it doesn't have the result objects yet?  if the later it could be that we are saving the objects after setting the task to complete since there is nothing transactional around the task persistence stuff